### PR TITLE
enabling connectors debugging via embed

### PR DESCRIPTION
**Pull request by tayrrible**
*Description here*

Fixes #**issue_number**

---

**Checklist**

Complete the checklist (and note appropriate exceptions) before the PR is marked ready-for-review.

- [ ] Changes are compatible[^1]
- [ ] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [ ] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]: It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]: Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]: Tick whichever testing boxes are applicable. If you are adding Manual Tests, please document the manual testing (extensively) in the Exceptions.

### CODE REVIEW

**Review by lennyburdette — APPROVED** [review]

File: apollo-router/templates/sandbox_index.html [hunk 1]
```diff
@@ -64,6 +64,7 @@ <h1>Welcome to the Apollo Router</h1>
             },
             hideCookieToggle: false,
             endpointIsEditable: false,
+            enableConnectorsDebugging: true,
```
Comment (lennyburdette) [inline apollo-router/templates/sandbox_index.html:67]: should the name match the other similar flag `initialRequestQueryPlan`?

Comment (tayrrible) [reply apollo-router/templates/sandbox_index.html:67]: @lennyburdette can do but would require refactoring & releasing studio code first

Comment (tayrrible) [reply apollo-router/templates/sandbox_index.html:67]: Would `initialRequestConnectorsDebugging` make sense?

Comment (lennyburdette) [reply apollo-router/templates/sandbox_index.html:67]: yeah that's what i was thinking. it's certainly not a requirement, but once this is baked into router's sandbox html, we'll need to support it in perpetuity.

**Pull request merged**